Mathlib/Topology/ContinuousOn.lean

```diff
@@ -1303,3 +1303,22 @@ theorem continuousWithinAt_prod_iff {f : α → β × γ} {s : Set α} {x : α}
       ContinuousWithinAt (Prod.fst ∘ f) s x ∧ ContinuousWithinAt (Prod.snd ∘ f) s x :=
   ⟨fun h => ⟨h.fst, h.snd⟩, fun ⟨h1, h2⟩ => h1.prod h2⟩
 #align continuous_within_at_prod_iff continuousWithinAt_prod_iff
+
+theorem IsClosedMap.image_closure_eq_of_continuous
```

Comment:
For discoverability, how about adding a comment above `image_closure_subset_closure_image` saying "See also `IsClosedMap.image_closure_eq_of_continuous`."

Reply:
Done. Actually your comment made me realize that I could use `image_closure_subset_closure_image` instead of `ContinuousOn.image_closure`, so I moved a few things around and changed order of some equalities.